lib/Conductor.js

```diff
@@ -178,6 +179,19 @@ function Conductor(config) {
     // now that dependencies are sorted, merge all the props, handlers, models.
     this._mergeDeps();
 
+    // this._handlers is an array of handler arrays, so loop in to the
+    // lowest level to check for each one being a function
+    _.map(this._handlers, function(handlers) {
```

Comment:
Ha, love that lodash will handle both arrays and keyed objects here in the same manner.